drivers/SmartThings/zigbee-lock/src/init.lua

```diff
@@ -57,6 +57,14 @@ local refresh = function(driver, device, cmd)
   device:refresh()
   device:send(LockCluster.attributes.LockState:read(device))
   device:send(Alarm.attributes.AlarmCount:read(device))
+  -- we can't determine from fingerprints if devices support lock codes, so
```

Comment:
Why have this in `refresh` instead of a less frequent handler like `configure`? Is the idea that this would this just make it easier for recovering since a refresh would be all that's needed?

Reply:
Right. Users should only need to pull down to recover.